data_model/src/transaction.rs

```diff
@@ -414,6 +416,7 @@ impl Txn for Transaction {
     FfiType,
     IntoSchema,
 )]
+#[allow(clippy::use_debug)]
```

Comment:
We have module level `!#[allow(clippy::use_debug)]` do we need this allow for individual struct.

Reply:
Indeed. Fixed